windows-vm-oms-extension/azuredeploy.parameters.json

```diff
@@ -0,0 +1,27 @@
+{
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "newStorageAccountName": {
+      "value": "GEN-UNIQUE"
+    },
+    "adminUsername": {
+      "value": "omsWindows"
+    },
+    "adminPassword": {
+      "value": "GEN-SSH-PUB-KEY"
```

Comment:
as [the documentation](https://github.com/azure/azure-quickstart-templates#parameters-file-placeholders) points out, you need to use `GEN-PASSWORD` when you need an azure-compatible password and `GEN-SSH-PUB-KEY` when you need an ssh public key. It looks like you meant to use `GEN-PASSWORD` here...